collections/_members/bart-kalanski.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+name: Bart Kalanski
+position: Contributor
+image_path:  https://pbs.twimg.com/profile_images/1271353670211375106/n4l3xu3k_400x400.jpg
```

Comment:
Can you please download the image and keep it locally like in other members profile?

Reply:
Ok sure I will commit again